cosmwasm/devnet/config/app.toml

```diff
@@ -44,7 +44,7 @@ halt-time = 0
 # It has no bearing on application state pruning which is determined by the
 # "pruning-*" configurations.
 #
-# Note: Tendermint block pruning is dependant on this parameter in conjunction
+# Note: Tendermint block pruning is dependant on this parameter in conunction
```

Comment:
Is that even a word?

Reply:
This file is a straight copy of https://github.com/terra-money/LocalTerra/tree/main/config 🙃